src/client/nativeCommunication.ts

```diff
@@ -46,8 +46,13 @@ function insertAds(): void {
     let adSlots = getAdSlots();
     if (adSlots.length > 0) {
         commercialClient.insertAdverts(adSlots);
-        const targetNode = document.querySelector('body') as Node;
-        const config = { attributes: true, childList: true, subtree: true };
+        const targetNode = document.querySelector('html') as Node;
+        const config: MutationObserverInit = {
+            attributes: true,
```

Comment:
I'm not sure this is needed with attributeFilter? https://developer.mozilla.org/en-US/docs/Web/API/MutationObserverInit/attributeFilter

Reply:
It's not needed for this to work, but if we only care about layout changes this could be a more performant observer than to observe every other attribute? Trying to meet you both in the middle 😅

Reply:
@faresite I may be misunderstanding this, but I *think* @webb04 is referring to the fact that the `attributes: true` property is superseded by the `attributeFilter` property? If this is correct then I think we can drop the `attributes` field here because it's applied implicitly by the `attributeFilter` field below. Or am I missing something? 🤔

Reply:
ah of course 🤦‍♀️  sorry @webb04 misunderstood you! you're both right.